UNIT_TESTS.md

```diff
@@ -0,0 +1,74 @@
+# Unit Tests
+
+Besides the code of the module itself, it is important to submit in addition a
+suite of tests. They guarantee your add-on remains functional when changes are
+applied to the code on which it depends. Once committed to GitHub, the unit
+tests are run automatically by the CI/CD pipeline every time the code in the
+repository is modified. Notifications are issued if a modification somehow
+impacts your add-on.
+
+## Requirements
+
+The unit tests depend on software that you must install in your system
+beforehand: Subversion and the GRASS development package. On a packaged operating
+system like Ubuntu or Debian Linux this can be achieved with:
+
+```bash
+sudo apt install subversion grass-dev
```

Comment:
Shall we omit "subversion" here? It may just add confusion and is usually no longer needed:

https://github.com/OSGeo/grass/pull/2895

Reply:
Reference to subversion removed in 0574f7287634ba26a4e334e07f326273b7153311